keylime/web_util.py

```diff
@@ -143,8 +143,7 @@ def echo_json_response(handler, code, status=None, results=None):
         return True
     if isinstance(handler, tornado.web.RequestHandler):
         handler.set_status(code)
-        handler.set_header('Content-Type', 'application/json')
-        handler.write(json_response)
```

Comment:
Since the `json_response` variable is only used if the handler is a `BaseHTTPRequestHandler` we should probably move it into that conditional. Should save one double JSON and UTF-8 encoding.

Reply:
fixed